web-client/src/presenter/actions/actionFactory.js

```diff
@@ -1,51 +1,6 @@
-import { getDefaultAttachmentViewerDocumentToDisplayAction } from './getDefaultAttachmentViewerDocumentToDisplayAction';
-import { getDefaultAttachmentViewerDocumentToDisplayAction as getDefaultAttachmentViewerDocumentToDisplayActionOld } from './getDefaultAttachmentViewerDocumentToDisplayAction.old';
-
-import { openCaseDocumentDownloadUrlAction } from './openCaseDocumentDownloadUrlAction';
-import { openCaseDocumentDownloadUrlAction as openCaseDocumentDownloadUrlActionOld } from './openCaseDocumentDownloadUrlAction.old';
-
-import { setAddEditUserCaseNoteModalStateFromDetailAction } from './TrialSessionWorkingCopy/setAddEditUserCaseNoteModalStateFromDetailAction';
-import { setJudgesCaseNoteOnCaseDetailAction } from './TrialSession/setJudgesCaseNoteOnCaseDetailAction';
-import { setViewerDocumentToDisplayAction } from './setViewerDocumentToDisplayAction';
-
-import { setAddEditUserCaseNoteModalStateFromDetailAction as setAddEditUserCaseNoteModalStateFromDetailActionOld } from './TrialSessionWorkingCopy/setAddEditUserCaseNoteModalStateFromDetailAction.old';
-import { setJudgesCaseNoteOnCaseDetailAction as setJudgesCaseNoteOnCaseDetailActionOld } from './TrialSession/setJudgesCaseNoteOnCaseDetailAction.old';
-import { setViewerDocumentToDisplayAction as setViewerDocumentToDisplayActionOld } from './setViewerDocumentToDisplayAction.old';
-
-import { generateTitleAction } from './FileDocument/generateTitleAction';
-import { generateTitleForPaperFilingAction } from './FileDocument/generateTitleForPaperFilingAction';
-
-import { isCodeDisabled } from '../../../../codeToggles';
+// import { isCodeDisabled } from '../../../../codeToggles';
```

Comment:
Could this file be deleted?

Reply:
I'm removing the commented-out import.  However, we're going to leave the `codeToggles.js` file in place because it's a piece of code that may be used in the near-future for larger feature-work.